metabase/metabase
# Indexed entity performance improvement

[request](https://metaboat.slack.com/archives/C04U3NW6UGP/p1686825717172239) 

### Description

Don't navigate away from the current model if the user searches for an item in that model

https://www.loom.com/share/1f3a7d1cf6814b3e82e4f89eef47b3a8

### How to verify

- activate a model index for a model based on a table in your db (like people)
- search for an item in that model in the app bar
- see that it navigates to the model
- search for another item in the same model from the app bar
- see that you don't navigate to a different page, the correct modal just pops up in the same model

TODO: tests

### Checklist
